src/pages/index.tsx

```diff
@@ -31,4 +33,15 @@ const Home: i.NextPageComponent = () => {
   );
 };
 
+export async function getStaticProps() {
+  const queryClient = new QueryClient();
+  await queryClient.prefetchQuery(['user', 'user_id'], () => fetchUser());
+
+  return {
+    props: {
+      dehydratedState: dehydrate(queryClient),
+    },
+  };
+}
```

Comment:
Ziet er wat boilerplaterig uit om elke keer een `queryClient` te instantiëren en `dehydrate` te gebruiken. Niet per se noodzakelijk om hier een abstractie voor te verzinnen op dit moment, maar misschien voor in de toekomst hier eens naar kijken.